docs/changelog.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+
+## 3.18.0 (2024-02-07)
+
+**🚀 New features**
+
+- Add HD viewpoints to treks/outdoorCours/POIs (#943)
+
```

Comment:
Ajouter : 
```
**🐛 Fixes**

- Fix header customization regression in version 3.17.0 (#1019)
- Fix scriptsHeader/Footer containing only one object, regression since 3.15.4 (#1029)
```

Dans maintenance, ajouter : 
```
- Update Cypress and development documentation (#1028)
```

Reply:
Ces fonctionnalités ont été ajouté lors de la release précédente https://github.com/GeotrekCE/Geotrek-rando-v3/releases/tag/v3.17.1

Reply:
Ah oui je me suis embrouillé !